### PR TITLE
Book controller and greeting view

### PR DESCRIPTION
**Pull request by alisonreynoso**
## Summary
- Created the BookController
- Created  the greeting.html
- Added thymeleaf dependency 
## Trello Card
https://trello.com/c/UMoNpows/17-crear-un-controller-y-una-vista-de-bienvenida

### CODE REVIEW

File: src/main/java/wolox/training/controllers/BookController.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
+package wolox.training.controllers;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+@Controller
+public class BookController {
+  @Controller
```
Comment (julianvasquez-wolox) [inline src/main/java/wolox/training/controllers/BookController.java]: Hi! It's not necessary to create a inner Greeting class 😊

Comment (alisonreynoso) [reply src/main/java/wolox/training/controllers/BookController.java]: Got it! I've pushed the changes :)

**Review by julianvasquez-wolox — DISMISSED** [review]
Niceee! :D

**Review by Jimenarosello — DISMISSED** [review]
Nice job! 😄

**Review by julianvasquez-wolox — APPROVED** [review]

**Review by Jimenarosello — APPROVED** [review]
🚀 😄

**Pull request merged**